livechat/agent/rtm/api/v33.py

```diff
@@ -16,24 +16,27 @@ def __init__(self, url: str):
 
     def open_connection(self,
                         origin: dict = None,
-                        ping_timeout: float = 3,
+                        ping_timeout: float = 30,
                         ping_interval: float = 5,
                         ws_conn_timeout: float = 10,
-                        keep_alive: bool = True) -> None:
+                        keep_alive: bool = True,
+                        response_timeout: float = 3) -> None:
```

Comment:
It seems strange to me that all values have the type marked as `float` and the default value as `int`. Since we are making changes in this area, I would consider changing all types to:
```suggestion
                        response_timeout: Union[float, int] = 3) -> None:
```
considering `(int or float)` in the docstring and https://websocket-client.readthedocs.io/en/latest/app.html#websocket._app.WebSocketApp.run_forever
> ping_interval (int or float), ping_timeout (int or float)

`ws_conn_timeout` should stay as `float` with `10.0` as default value.

Reply:
Therefore I have chosen float to be consistent with others. Anyway, I will change all types to Union[ float, int].